misc/src/main/java/com/example/snippets/ActivityEmbeddingJavaSnippets.java

```diff
@@ -336,6 +319,23 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             super.onCreate(savedInstanceState);
             setContentView(R.layout.activity_main);
 
+            // [START android_activity_embedding_splitAttributesBuilder_java]
+            SplitAttributes.Builder splitAttributesBuilder = new SplitAttributes.Builder()
```

Comment:
Just to note, we generally create functions that separate out each snippet logically, otherwise reading the file in its entirety is a bit difficult if they are all in the same function.

Reply:
I reorganized the files to group related snippets.